spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+require 'formatter'
+
+include Formatter
+
+describe Menu do
+
+  let(:dish_klass){ double(:dish_klass, new: dish) }
+  let(:dishes_str){ "Burger,5.99/" }
+  let(:dish){ double(:dish, name: 'Burger', price: 5.99) }
+
+  subject(:menu){ Menu.new(dish_klass, dishes_str) }
+
+  context '#formatted_dishes' do
+    it 'returns a formatted list of dishes' do
+      name, price = dish.name, pound_str(dish.price)
```

Comment:
Style/ParallelAssignment: Do not use parallel assignment. (https://github.com/bbatsov/ruby-style-guide#parallel-assignment)